NitroxClient/Communication/Packets/Processors/PlayerJoinedMultiplayerSessionProcessor.cs

```diff
@@ -23,7 +23,9 @@ public override void Process(PlayerJoinedMultiplayerSession packet)
 
             playerManager.Create(packet.PlayerContext);
 
-            Log.InGame($"{packet.PlayerContext.PlayerName} joined the game.");
+            string PlayerJoinedText = $"{packet.PlayerContext.PlayerName} {Language.main.Get("Nitrox_PlayerJoined")}";
+            Log.InGame(PlayerJoinedText);
```

Comment:
```suggestion
            Log.InGame($"{packet.PlayerContext.PlayerName} {Language.main.Get("Nitrox_PlayerJoined")}");
```

Reply:
The playername should be a parameter in the localization string like in other messages imo.